src/main/js/helpers/legend.js

```diff
@@ -392,19 +394,36 @@ const loadPieLegendItem = (legendSVG, dataTarget, { hoverHandler }, config) => {
 const getDefaultLegendOptions = (graphConfig, dataTarget) => {
     const legendOptions = getDefaultValue(dataTarget.legendOptions, {
         showShape: true,
-        showLine: false,
+        showLine: dataTarget.showShapes === false,
```

Comment:
how does this impact graphs types that are not lines like bar charts.  I am seeing the line displayed in bar and combination graphs
![image](https://user-images.githubusercontent.com/44301749/92745320-de897780-f347-11ea-8f00-6a80771cf3b4.png)

Reply:
For some reason the checkout of your branch did not have all the latest changes.  After resetting, the axis label shapes/lines are appearing or not appearing as expected.